### DISCUSSION TITLE
Aligning to a single reference image

### DISCUSSION THREAD

**Discussion by mariakesa** [voxelmorph/voxelmorph#204] [Q&A] [answered]
Hello:-)

Is it possible to align a set of images to a single reference image using voxel morph? How would the training code look for that?

Thank you so much for  your work and for the wonderful software!:-)

Maria

**Comment by adalca** [comment, marked as answer]
Hi @mariakesa 

yes, this is in fact the setting of our first experiments https://arxiv.org/abs/1809.05231 

[train.py](https://github.com/voxelmorph/voxelmorph/blob/master/src/train.py) should be a good guideline.

What are you aligning?

Adrian

**Comment by mariakesa** [comment]
Great thank you! Sorry for asking stupid questions! Thank you for the great Kaggle tutorial:-)

I am starting my PhD in Neuroscience working with massive whole-brain scale experiments on larval zebrafish. Something like this https://www.youtube.com/watch?v=eKkaYDTOauQ One experiment is 50 GB of data:-D Going to try to align the calcium imaging experiments with a reference zebra fish atlas. 

Thank you once again:-)

Maria

**Comment by adalca** [comment]
@mariakesa  that sounds pretty interesting. Let me know how it goes!
